src/main/java/com/github/calebwhiting/runelite/plugins/actionprogress/detect/SmithingDetector.java

```diff
@@ -90,7 +90,11 @@ public void onVarbitChanged(VarbitChanged evt)
 	{
 		if (evt.getVarbitId() == VAR_SELECTED_SMITHING_INDEX) {			
 			int index = evt.getValue();
-			smithingItemid = indexToItemId.get(index);
+			Integer itemID = indexToItemId.get(index);
```

Comment:
I just merged a PR where I fixed this. I suggest you revert that to solve the conflict

Reply:
Just saw the update. Resolving this.